src/qt/bitcoingui.cpp

```diff
@@ -454,6 +454,46 @@ void BitcoinGUI::createMenuBar()
     appMenuBar = menuBar();
 #endif
 
+    // Add chain selection submenu
+    QAction* chain_selection_action{nullptr};
+    QMenu* chain_selection_menu{nullptr};
+    chain_selection_action = new QAction(tr("&Switch chain"), this);
+    chain_selection_action->setStatusTip(tr("Restart application using a different (test) network"));
+    chain_selection_menu = new QMenu(this);
+
+    connect(chain_selection_menu, &QMenu::aboutToShow, [this, chain_selection_menu] {
+        chain_selection_menu->clear();
+        const std::vector<std::pair<QString, QString>> chains = {{"main", "&Bitcoin"}, {"test", "&Testnet"}, {"regtest", "&Regtest"}, {"signet", "&Signet"}};
```

Comment:
In commit "gui: toggle chain via menu" (12431e0d95157b4df18912b38735acdc927c9762)

Interesting the text is is "Bitcoin" instead of "Mainnet". I think that it is good because it implies the other networks are not the Bitcoin network. Another way to do this while being a little more consistent could be to make the choices "Mainnet (Bitcoin)" / "Testnet" / "Regtest" / "Signet"

Reply:
I noted this as well - IMO "Bitcoin" is a better way to go - "Mainnet" may actually confuse new users at this point.